src/main/java/io/fabric8/maven/docker/util/Logger.java

```diff
@@ -26,12 +26,13 @@
     void info(String format, Object ... params);
 
     /**
-     * Verbose message
+     * Verbose message for build
      *
+     * @param logVerboseCategory debug level for logging
      * @param format verbose message format
      * @param params parameter for formatting message
      */
-    void verbose(String format, Object ... params);
+    void verbose(AnsiLogger.LogVerboseCategory logVerboseCategory, String format, Object ... params);
```

Comment:
We should move the category to this interface. It's a bad smell to have the interface to reference to one of its implementations.